sdcm/cluster.py

```diff
@@ -5078,7 +5078,7 @@ def get_node_ip_list(verification_node):
 
         missing_host_ids = verification_node.raft.get_diff_group0_token_ring_members()
 
-        if not missing_host_ids:
+        if not missing_host_ids and ComparableScyllaVersion(verification_node.scylla_version) <= "2025.1":
```

Comment:
Could we use some feature flag instead?

Reply:
I just saw doc, saying about `GROUP0_LIMITED_VOTERS` flag.
Should use it for clarity (and upgrade tests proper work)

Reply:
I'm ok work with version. Because in fact the upgrade procedure is changed(node checks) and this feature will be enabled by default. 

I just not happy with this amount of similar if's. @aleksbykov, could you please check if you can do something with that?